ocaml/libs/sexpr/test/dune

```diff
@@ -1,4 +1,4 @@
 (test
  (name test_sexpr)
  (modules test_sexpr)
- (libraries sexpr astring rresult qcheck alcotest threads))
+ (libraries sexpr astring rresult qcheck-core alcotest threads))
```

Comment:
```suggestion
 (package sexpr)
 (libraries sexpr astring rresult qcheck-core alcotest threads))
```

Reply:
Ah that is what you meant by package :)

Reply:
The signoff that gets added by the github UI is not ideal (@users.noreply.github.com), but well it still uniquely identifies us, so probably good enough.

Reply:
The commit got lost :(